app/uk/gov/hmrc/merchandiseinbaggage/controllers/Navigator.scala

```diff
@@ -40,11 +41,13 @@ object Navigator {
 
   def nextPage: Map[String, Call] = Map(
     AgentDetailsController.onPageLoad().url -> EnterAgentAddressController.onPageLoad(),
-    EnterEmailController.onPageLoad().url   -> JourneyDetailsController.onPageLoad()
+    EnterEmailController.onPageLoad().url   -> JourneyDetailsController.onPageLoad(),
+    EoriNumberController.onPageLoad().url   -> TravellerDetailsController.onPageLoad(),
   )
 
-  def nextPageWithAnswer: Map[String, YesNo => Call] = Map(
-    CustomsAgentController.onPageLoad().url -> customsAgent
+  def nextPageWithAnswer[T]: Map[String, T => Call] = Map(
+    GoodsDestinationController.onPageLoad().url -> goodsDestination,
+    CustomsAgentController.onPageLoad().url     -> customsAgent
   )
 
   def nextPageWithIndex: Map[String, (YesNo, JourneyType, Int) => Call] = Map(
```

Comment:
all the above defs could be vals or lazy vals ?

Reply:
sure, done